tests/test_cfg_end_to_end_tests.py

```diff
@@ -29,21 +30,72 @@
 
 DATASET_CONFIGS_ROOT = Path(__file__).parents[1] / "dataset_configs"
 
-def data_check_fn_generic(raw_data_dir: str, file_name: str, **kwargs) -> None:
```

Comment:
This function was added quite recently. Do you intend to get rid of it? Fine with me, just want to check it's not an accident.